apps/docs/components/ui/pre/Pre.tsx

```diff
@@ -1,19 +1,19 @@
-import React, { type HTMLAttributes } from "react";
-import { CopyButton } from "@/components/Utils/CopyButton/CopyButton";
-import { LangIcon } from "@/components/Pre/LangIcon/LangIcon";
+import type { HTMLAttributes } from "react";
```

Comment:
Should we always import types first? What's our rule on this? I usually importe them after.

Reply:
I usually import all the external libraries first, then the internal components and finally the css. 
For types, I often import them with external components. 

**Example**

```jsx
// import external components
// import types of external components

// import local component with alias

// import utilities and helper

// import style.css
```